build.gradle.kts

```diff
@@ -468,8 +468,10 @@ tasks.register("createDmg") {
 fun genBuildInfo(buildInfoFilePath: String) {
   val now = Date()
   val nowIso = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ").format(now)
-  val branchName = runCommand(listOf("git","rev-parse", "--abbrev-ref", "HEAD"), "Failed getting branch name.")
-  val branchLastCommitHash = runCommand(listOf("git","rev-parse", "--short=8", "HEAD"), "Failed getting last commit has.")
+  val branchName = runCommand(listOf("git", "-C", "$projectDir", "rev-parse", "--abbrev-ref", "HEAD"),
```

Comment:
Please remove quotes from `"$projectDir"` (multiple places).

Reply:
I still haven't followed up here. So I will post responses here and set up a new PR.

I can't remove those quotes. `projectDir` is a File, not a String. I can remove the quotes if I write `projectDir.toString()`. I think I'll stick with quotes. But I should have written `"${projectDir}"` so I'll go with that.